service/src/main/java/com/ge/predix/acs/policy/evaluation/cache/PolicyEvaluationCache.java

```diff
@@ -29,24 +29,26 @@
 
     void set(PolicyEvaluationRequestCacheKey key, PolicyEvaluationResult value);
 
-    void setResourceTranslation(String zoneId, String fromResourceId, String toResourceId);
-
-    void setResourceTranslations(String zoneId, Set<String> fromResourceIds, String toResourceId);
-
     void reset();
 
     void reset(PolicyEvaluationRequestCacheKey key);
 
     void resetForPolicySet(String zoneId, String policySetId);
 
+    void setPolicySetIfNotExists(String zoneId, String policySetId);
```

Comment:
I agree that setPolicySetIfNotExists, setResourceIfNotExists and setSubjectIfNotExists should not be part of the cache interface (because these are only used internally in the AbstractPolicyEvaluationCache). But why should setIfNotExists be apart of the interface if it is also only use internally?